civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -155,5 +169,29 @@ function filterTable(statusId) {
     $filtersDropdown.on('change', function (e) {
       filterTable($(this).val());
     });
+
+    // Trigger event to trigger appDocuments forom T&A
+    function taInit() {
+      var detail = {
+        'app': 'appDocuments',
+        'module': 'tnadocuments'
+      };
+
+      document.dispatchEvent(typeof window.CustomEvent == "function" ? new CustomEvent('taInit', {
+        'detail': detail
+      }) : (function(){
+        var e = document.createEvent('CustomEvent');
+        e.initCustomEvent('taInit', true, true, detail);
+        return e;
+      })());
+    };
+
+    taInit();
+
+    document.addEventListener('taReady', function(){
+      taInit();
+    });
   }(CRM.$));
 </script>
+
+<script src="<?php echo getModuleUrl('uk.co.compucorp.civicrm.tasksassignments').'js/dist/tasks-assignments.min.js';?>"></script>
```

Comment:
I can use `civicrm_resources_load(...)` here as order of execution of scripts is important. As `civicrm_resources_load(...)` tries to load the scripts before the inline scripts are loaded, thus the settings variables 
```

    CRM.contactId = Drupal.settings.contactId;
    CRM.adminId = Drupal.settings.adminId;
    CRM.Tasksassignments = Drupal.settings.Tasksassignments;
```
will not be available for `tasks-assignments.min.js` causing following exception:
<img width="1367" alt="screen shot 2017-05-29 at 2 02 09 pm" src="https://cloud.githubusercontent.com/assets/6307362/26541931/92d80180-4477-11e7-8e77-7e360d7a1841.png">